### PR TITLE
Rename k8s ingress and backend config name

### PR DESCRIPTION
**Pull request by blackzlq**
Fix the naming mess for IAP.
k8s ingress should be named as proxy-public
backend service for now name it as iap-config-default

Tested-by: zlq

Tested:
Sandbox

### CODE REVIEW

File: applications/jupyter/main.tf [hunk 1]
```diff
@@ -81,15 +81,15 @@ module "jupyterhub" {
...
-  url_domain_addr         = var.url_domain_addr
-  url_domain_name         = var.url_domain_name
-  members_allowlist       = var.members_allowlist
-  depends_on              = [module.gcs]
+  add_auth                  = var.add_auth
+  brand                     = var.brand
+  support_email             = var.support_email
+  client_id                 = var.client_id
+  client_secret             = var.client_secret
+  k8s_ingress_name          = var.k8s_ingress_name
```
Comment (umeshkumhar) [inline applications/jupyter/main.tf:89]: this is not exactly a ingress.. this is the ingress - default backend k8s_service.

this is configured as default traffic that will be sent to this service.. (proxy-public, i.e. of jupyter)

Comment (imreddy13) [reply applications/jupyter/main.tf:89]: proxy-public is not an ingress, it is a k8s service. i think the previous name is correct

File: applications/jupyter/main.tf [hunk 2]
```diff
@@ -81,15 +81,15 @@ module "jupyterhub" {
...
+  k8s_backend_config_name   = var.k8s_backend_config_name
```
Comment (imreddy13) [inline applications/jupyter/main.tf:90]: this can be k8s_ingress

**Comment by imreddy13** [comment]
not convinced why we need this change

**Pull request closed without merging**